packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -124,6 +124,41 @@ QUnit.module('rendering', () => {
         }
     });
 
+    QUnit.test('multiview items.visible option should always be default to true', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1' }
+            ]
+        });
+        const $items = $multiView.find(toSelector(MULTIVIEW_ITEM_CLASS));
+
+        assert.notOk($items.eq(0).hasClass(MULTIVIEW_ITEM_HIDDEN_CLASS), 'first item is visible');
+    });
+
+    QUnit.test('multiview should not render items those that are option visible is false', function(assert) {
```

Comment:
[naming]
Same idea here. U say "should not render", but it's actually rendered but hidden with css
```suggestion
    QUnit.test('items with visible=false should be hidden', function(assert) {
```

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)